website/docs/guides/dependencies.md

```diff
@@ -58,6 +58,19 @@ fn main() -> felt252 {
 }
 ```
 
+## Development dependencies
+
+You can add a `[dev-dependencies]` section to your Scarb.toml whose format is equivalent to `[dependencies]`:
+
+```toml
+[dev-dependencies]
+tempdir = "0.3"
+```
+
+Dev-dependencies are not used when compiling a package for building, but are used for compiling tests.
+
+These dependencies are not propagated to other packages which depend on this package.
+
```

Comment:
In general, we should keep `guides/*` short, and `reference/*` thorough and precise, so I suggest moving this explanation to `reference/specifying-dependencies#Development dependencies `